include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_merge.h

```diff
@@ -96,8 +97,8 @@ __serial_merge(const _Rng1& __rng1, const _Rng2& __rng2, _Rng3& __rng3, _Index _
     _Index __rng1_idx = __start1;
     _Index __rng2_idx = __start2;
     _Index __rng3_idx = __start3;
-    
-    #pragma unroll _UnrollFactor
+
+#pragma unroll _UnrollFactor
```

Comment:
`#pragma` is not portable.. OneDPL has a macro
// Portability "#pragma" definition
```
#ifdef _MSC_VER
#    define _ONEDPL_PRAGMA(x) __pragma(x)
#else
#    define _ONEDPL_PRAGMA(x) _Pragma(#    x)
#endif

```

Reply:
Could you please explain how to apply this for unroll?

Reply:
We made decision together with @dmitriy-sobolev to remove `unroll` here at all: no perf profit has been detected.